google-cloud-tools-plugin/resources/messages/CloudToolsBundle.properties

```diff
@@ -132,6 +132,7 @@ appengine.deployment.error.invalid.cloudsdk=Invalid Cloud SDK directory path.
 appengine.deployment.flexible.nogaesupport=No modules with Google App Engine support found. Please provide your configuration files
 appengine.deployment.error.cancelled=Deployment process was cancelled.
 appengine.facet.converter.description=Deprecated Google App Engine facets will be replaced with the latest version
+appengine.deploy.run.config.converter.description=Convert deprecated plugin deployment run configurations to work with the new plugin
```

Comment:
actually it might be shown in the bottom left messaging in the ide when the converter window pops up. either way, I updated it